assets/js/dist/fade.js

```diff
@@ -1,40 +1,38 @@
 "use strict";
 
-$(document).ready(function () {
-  var mobileOffset = 270; // Start with only work section showing
+var mobileOffset = 270; // Start with nav-work button highlighted and only work section showing
```

Comment:
This is currently creating an implicit dependency between this file and whatever CSS sets the height. Is there some way you could reorganize this so that there is a single source of truth?

Perhaps you could derive this value here in JS?